Update workspaces.rb

ShaunGrey opened the pull request:
created a "update_workspace" function in workspace.rb to allow updating workspaces

ShaunGrey committed:
Update workspaces.rb

created a "update_workspace" function in to allow updating workspaces

kanet77 commented:
Thanks for adding this functionality. Could you add a test to `workspaces_spec.rb`?

If you don't get to it, I'll write a test the next time I can work on this in about two weeks.

kanet77 commented:
By the way, if you merge `master` into your PR branch, the tests will pass and Travis CI builds will complete.

ShaunGrey committed:
Add update_workspace test case to worspace_spec.rb

ShaunGrey committed:
Update workspaces_spec.rb

removed some keys in the update_workspace method that will fail on a non-paid workspace

ShaunGrey committed:
Update workspaces_spec.rb

fixed typo that was putting keys in as symbols

ShaunGrey committed:
Update reportsv2.rb

removed unnecessary colons in the "project" def's return parameters

ShaunGrey committed:
Update reportsv2.rb

removed unnecessary colons in the "report" def's return parameters

ShaunGrey commented:
I believe the test is all added in now! Let me know if it's no good, i'm kinda new to this.
